modules/ROOT/pages/authentication-authorization/property-based-access-control.adoc

```diff
@@ -0,0 +1,58 @@
+:description: How to use Cypher to manage property-based access control on graphs.
+
+////
+[source, cypher, role=test-setup]
+----
+CREATE ROLE regularUsers;
+----
+////
+
+[role=enterprise-edition aura-db-enterprise]
+[[property-based-access-control]]
+= Property-based access control
+
+It is possible to create `READ` privileges that are based on properties on nodes.
+To specify which property to base the `READ` privilege the `pattern` syntax described bellow is used,
+for more information about `READ` privilege syntax see xref:authentication-authorization/privileges-reads.adoc[read privilege] page.
+Adding property-based access control can incur a significant performance overhead in certain scenarios,
+see xref:authentication-authorization/limitations.adoc#property-based-access-control-limitations[Limitations] for more detailed information.
+To reduce the performance impact, we recommend using Block Storage as this is better optimised for the kind of read required for the resolution of property-based privileges.
+
+Pattern syntax:
+[source, syntax, role="noheader"]
+----
+([var][:label["|" ...]] "{" property: value "}")
+| (var[:label["|" ...]]) WHERE [NOT] var.property { = value | <> value | IS NULL | IS NOT NULL }
+| (var[:label["|" ...]] WHERE [NOT] var.property { = value | <> value | IS NULL | IS NOT NULL } )
+----
+[NOTE]
+====
+For more details about the syntax descriptions, see xref:database-administration/syntax.adoc[].
+====
+
+
+For example, to grant role `regularUser` the ability to `READ` property `address` on nodes with labels `Email` or `Website` where the domain is `exampledomain.com`:
+[source, syntax, role="noheader"]
+----
+GRANT READ { address } ON GRAPH * FOR (n:Email|Website) WHERE n.domain = 'exampledomain.com' TO regularUsers
+----
+if preferred the same can be expressed using this syntax as well:
+[source, syntax, role="noheader"]
+----
+GRANT READ { address } ON GRAPH * FOR (:Email|Website {domain: 'exampledomain.com'}) TO regularUsers
+----
+
+
+Granting `TRAVERSE` privilege on nodes with label `Email` where property `classification` is `NULL` to role `regularUsers`:
+[source, syntax, role="noheader"]
+----
+GRANT TRAVERSE ON GRAPH * FOR (n:Email) WHERE n.classification IS NULL TO regularUsers
+----
+
+To grant read privilege only for nodes that have a specific value on a property.
```

Comment:
this sentence seems out of place?

Reply:
removed. I think it just was a copy paste mistake.